tests/ctst/world/Zenko.ts

```diff
@@ -1,15 +1,94 @@
-import { setWorldConstructor, World } from '@cucumber/cucumber';
+import { setWorldConstructor, World, IWorldOptions } from '@cucumber/cucumber';
 import axios, { AxiosRequestConfig, AxiosResponse, Method, AxiosRequestHeaders } from 'axios';
-import { aws4Interceptor } from 'aws4-axios'
-
-import { CacheHelper, cliModeObject, Constants, IAM, IAMUserPolicy, STS, SuperAdmin, Utils, } from 'cli-testing';
-import { extractPropertyFromResults } from "../common/utils";
+import { aws4Interceptor } from 'aws4-axios';
+import {
+    CacheHelper,
+    ClientOptions,
+    cliModeObject,
+    Constants,
+    IAM,
+    IAMUserPolicy,
+    STS,
+    SuperAdmin,
+    Utils,
+    AWSCliOptions,
+} from 'cli-testing';
+import { Credentials } from 'aws4-axios';
+import { extractPropertyFromResults } from '../common/utils';
 import qs = require('qs');
 
-interface AwsCliObjectParameters {
+export interface AwsCliObjectParameters {
     [key: string]: number | string | undefined | object;
 }
 
+export interface AWSVersionObject {
+    Key: string;
+    VersionId: string;
+}
+
+export interface NotificationDestination {
+    destinationName: string;
+    topic: string;
+    hosts: string;
+}
+export interface Saved {
+    notificationDestinations: NotificationDestination[],
+    notificationsPerDestination: { [key: string]: string[] },
+    bucketName: string,
+    notificationEventType: string,
+    objectNamePrefix: string,
+    filterType: string,
+    objectName: string,
+    objectNameSufix: string,
+    [key: string]: unknown
+}
+
+interface Parameters {
+    [key: string]: unknown;
+}
+
+export interface Result {
+    err: null;
+    stdout: string
+}
+
+interface Role {
+    Name: string;
+    Arn: string;
+}
+
+interface Account {
+    id: string;
+    Arn: string;
+    name: string;
+    Roles: Role[];
+}
```

Comment:
Agreed, this is what I was expecting as feedback to this PR. 

I'm not sure I understand your comment on the second paragraph however; are you suggesting how we should handle adding these types to CTST?

Reply:
On this topic, I have identified (atleast for this file) the types that need defining in the cli-testing project:
- CacheHelper.parameters
- CacheHelper.ARWWI (list containing ClientOptions['AssumedSession'] type objects)
- Account (SuperAdmin.ts->getAccount return value)
- add Roles array to Account type in utils.ts
- I currently use the 'Result' interface to evaluate to the { err: null; stdout: string } type returned by the cli IAM APIs: it might be worth adding that to cli-testing (make buildAwsCliCommand return Result)

Reply:
> I'm not sure I understand your comment on the second paragraph however; are you suggesting how we should handle adding these types to CTST?

It's just that some of CTST files are generated and maybe not typed as you need, in this case, it's important to handle it! The good way is to update the generator scripts to include the right types you want, if this is needed.